less/index.less

```diff
@@ -65,112 +66,254 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-    width: 880px;
+    max-width: 880px;
+		width: 100%;
     margin: auto;
 }
 
+/*=============== Variables */
```

Comment:
Here is a good spot for global styles (header color, nav color, etc.) which have other variables assigned to them, like you did with `@hover-color`, and then use the color website to better name the colors: http://chir.ag/projects/name-that-color/